src/concepts/choosing_a_dataset.md

```diff
@@ -8,19 +8,23 @@ This document will help you find the best data source for a given analysis. It f
 
 If you want to know whether a causal link occurs between two events, you can learn more at [tools for experimentation](../tools/experiments.md).
 
-## Table of Contents
+There are two types of datasets that you might want to use: those based on raw pings and those derived from them.
 
-<!-- toc -->
+## Raw Ping Datasets
 
-## Raw Pings
+We receive data from Firefox users via **pings**: small JSON payloads sent by clients at specified intervals.
+There are many types of pings, each containing different measurements and sent for different purposes.
 
-{{#include ../datasets/ping_intro.md}}
+These pings are then [aggregated into ping-level datasets](../cookbooks/bigquery/querying.md#structure-of-ping-tables-in-bigquery) that can be retrieved using BigQuery.
+Pings can be a bit difficult to work with in their raw form and there are many pitfalls that are easy to fall into when using them naively.
+Where possible, you should use a derived dataset to answer your question.
```

Comment:
Do we still believe this? `main_summary` is described as deprecated; _that_ page tells you to use `telemetry.main`. If we don't have a practice of adding new columns to `clients_daily` and `clients_last_seen` then there are classes of analysis they won't support.

I also think the "raw ping" phraseology is a throwback to a time where the storage format and access semantics were meaningfully different from the derived datasets; maybe this is out of scope for this change but it probably makes sense to just call them "ping tables" now.

Reply:
I'd probably describe `main_summary` less as a derived dataset and more as a precursor to the ping tables. Other than that, I agree with you that the wording could be better, let's try to do that while we're here.